### ISSUE TITLE
prim::ConstantChunk derivative formula doesn't handle undefined inputs

### ISSUE THREAD

**Issue by apaszke** [pytorch/pytorch#14484]
It's a multi-output node, so it has to handle the case when not all outputs get well defined gradients (e.g. when some outputs are unused).

**Comment by janeyx99** [comment]
TorchScript has been deprecated in favor of torch.export as of 2.11. https://pytorch.org/blog/pytorch-2-11-release-blog/ Closing this issue as we are not further developing on it--please ping if you believe this issue was erroneously closed.
